src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/Orchestrators/SendPaymentsForAccountLegalEntityOrchestrator.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.EmployerIncentives.Functions.PaymentsProcess.Activities;
+
+namespace SFA.DAS.EmployerIncentives.Functions.PaymentsProcess.Orchestrators
+{
+    public class SendPaymentsForAccountLegalEntityOrchestrator
+    {
+        private readonly ILogger<SendPaymentsForAccountLegalEntityOrchestrator> _logger;
+
+        public SendPaymentsForAccountLegalEntityOrchestrator(
+            ILogger<SendPaymentsForAccountLegalEntityOrchestrator> logger)
+        {
+            _logger = logger;
+        }
+
+        [FunctionName(nameof(SendPaymentsForAccountLegalEntityOrchestrator))]
+        public async Task RunOrchestrator([OrchestrationTrigger] IDurableOrchestrationContext context)
+        {
+            var accountLegalEntityCollectionPeriod = context.GetInput<AccountLegalEntityCollectionPeriod>();
+
+            if (!context.IsReplaying) 
+                _logger.LogInformation("SendPaymentsForAccountLegalEntityOrchestrator started for Account Legal Entity: {accountLegalEntityCollectionPeriod}", accountLegalEntityCollectionPeriod);
+
+            await context.CallActivityAsync(nameof(SendClawbacksForAccountLegalEntity), accountLegalEntityCollectionPeriod);
```

Comment:
I think it would be better to await a Task.WhenAll here on both tasks.  Then all errors are propagated at once

Reply:
Updated